packages/nextjs/components/scaffold-eth/Contract/InheritanceTooltip.tsx

```diff
@@ -0,0 +1,14 @@
+import { InformationCircleIcon } from "@heroicons/react/24/solid";
+
+export const InheritanceTooltip = ({ inheritedFrom }: { inheritedFrom?: string }) => (
+  <>
+    {inheritedFrom && (
+      <span
+        className={`cursor-help tooltip tooltip-top tooltip-accent px-2 before:content-[attr(data-tip)]`}
```

Comment:
Not sure about `cursor-help` 😅 `cursor-pointer` looks good 

Maybe we also `break-normal` class? 

| Before | After | 
| ------- | ----- |
| ![Screenshot 2023-10-26 at 9 13 51 AM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/d973cad1-e546-4d89-b5e0-9420a4c37921) |  ![Screenshot 2023-10-26 at 9 13 30 AM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/94fca309-7bf0-447d-be8b-c532e85f5d68) |

I mean since it is a tooltip even if it gets a bit longer it's fine I think

Also `before:content-[attr(data-tip)]` why do we need it?  I tried removing it and everything looks fine but I might be missing something

Reply:
I agree that `cursor-help` looks weird. I can remove it. But if we do `cursor-pointer` wouldn't it seem like it's clickable? I'm not sure that's good.

Thank you for the tips on Tailwind/CSS. I'm still learning. I will add `break-normal` and `before:content-[attr(data-tip)]` was from copy/paste here: https://github.com/scaffold-eth/scaffold-eth-2/blob/e88419a97baedd81fceb37a2553bf49e80ad981b/packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx#L104

Reply:
>  But if we do cursor-pointer wouldn't it seem like it's clickable? I'm not sure that's good.

Yeah, I think we can just keep the default one (remove `cursor-help` and it should use the default).

- You can remove the template string for the className (since we are not using any vars)

Reply:
Fixed in: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/a4e3e42a3a058c3b4129bdc5b881a82596d56ffa

Reply:
We don't need the curly braces either!

Reply:
> and before:content-[attr(data-tip)] was from copy/paste here:

Ohh actually we used `before:content-[attr(data-tip)]` their because we wanted to ovveride default `tooltip` styles like shifting it to right by `-10px` etc....If we didn't override  it, it would break responsiveness on small screens : 
![Screenshot 2023-10-26 at 4 12 02 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/02460294-86c8-4ddf-9aff-162eeb811dca)

---

For this `InheritanceTooltip` we don't need `before:content-[attr(data-tip)]` since we not overriding any default styles which `tooltip` class adds 🙌

Also just to handle any edge cases on small screen if path is too large for inherited source we could do `md:break-normal` (so by default it breaks on smaller screens and on larger screens its one-liner)

So finally it will be: `className="tooltip tooltip-accent px-2 md:break-normal"`

Reply:
Fixed